packages/vertica-nodejs/test/integration/client/configuration-tests.js

```diff
@@ -16,27 +16,27 @@ suite.test('default values are used in new clients', function () {
     user: process.env.USER,
     database: undefined,
     password: null,
-    port: 5432,
+    port: 5433,
     rows: 0,
     max: 10,
     binary: false,
     idleTimeoutMillis: 30000,
     client_encoding: '',
-    ssl: false,
+    tls_mode: 'disable',
     application_name: undefined,
     fallback_application_name: undefined,
     parseInputDatesAsUTC: false,
   })
 
   var client = new vertica.Client()
   assert.same(client, {
-    user: process.env.USER,
-    database: process.env.USER,
     password: null,
-    port: 5432,
+    port: 5433,
```

Comment:
Isn't it bad to have our tests hard code the port?   I would think we would want to depend on the V_* environment variables.

Reply:
This test is meant to test what the default values are, but you are right that it doesn't play nice with our environment variables. Not sure what to do with this one.

Reply:
The problem is that if environment variables are set then that client created with no config object is going to use the environment variables instead of the defaults (which may or may not be the same, probably will be most of the time). However starting on line 15 we just asserted that the all the defaults are the same as what was hard coded, is there any real benefit from having this client comparison assert that we don't already essentially get from asserting the defaults are what we expect they are? If not I would say just remove that part of the test

Reply:
Good point, removing.